### ISSUE TITLE
Trigger grass cutter & Map objects destroying

### ISSUE THREAD

**Issue by SyMP2005** [Theseus-Aegis/Olympus#28]
**Grass cutter**
I've been looking to use a more efficient way to remove grass and stumbled upon this topic:
[https://forums.bistudio.com/topic/143399-cutter-grass-inside-a-triggermarker/](url)  

-As far as I understand that person's intention is also to create a grass cutter based on an area, opposed to the semi-retarded 10x10m circles BI provides.
Overall, if this script is feasible,it would really help cutting the grass from courses / areas. Without having to use an additional 50 objects (BI Grass cutters).  

**Map Object destroying**
While I stumbled upon the above I found this as well:
[https://www.youtube.com/watch?v=Nvf-ZsG4TnA](url)  

-Basically he's removing "destroyable" map objects, on init (in this case the bush). It seems to work as intended and it could be very useful to get rid of those hideous trees on the basic rifle course. However he's doing it with `nearestObjects`. Which, iirc, makes @jonpas cringe.
Can this be done in another way, coding wise, and is it a viable to use it in general?

**Comment by jonpas** [comment]
> Grass cutter

This script spawns grass cutters, I don't see how it's any better, it just adds overhead.

> Map Object destroying

~~Using `nearestObjects` is fine, but not for large distances and should be avoided whenever possible. It's viable as long as you keep it to small distances.~~
Checked closely, it's using `POSITION nearestObject ID`, where ID is the ID of the map object. I am not sure, but IIRC you can also synchronize those objects to triggers and then have a nicer way of handling them.
